.github/workflows/build.yaml

```
@@ -172,12 +240,21 @@ jobs:
       skip_upload_pkgs: libcuvs-template
   rust-publish:
     needs: [rust-build, upload-conda]
-    secrets: inherit
+    permissions:
+      contents: read
+    secrets:
+        GPU_CRATES_TOKEN: ${{ secrets.GPUTESTER_CRATES_TOKEN }}
```

actionlint (1.7.12)

[error] secret "GPU_CRATES_TOKEN" is not defined in "./.github/workflows/publish-rust.yaml" reusable workflow. no secret is defined

(workflow-call)